HAL/pico/include/config_defaults.hpp

```diff
@@ -166,7 +179,7 @@ const Config default_config = {
     .default_backend_config = 1,
     .default_usb_backend_config = 1,
     .melee_options = {
-        .crouch_walk_os = false,
+        .crouch_walk_os = false, 
```

Comment:
```suggestion
        .crouch_walk_os = false,
```
Please remove trailing whitespace (ideally you'd run clang-format on everything as well)